contribute/architecture/frontend-data-requests.md

```diff
@@ -0,0 +1,51 @@
+# Data requests
+
+[BackendSrv is the core data request service](https://grafana.com/docs/grafana/latest/packages_api/runtime/backendsrv) used by Grafana. This documentation highlights the high level concepts used by `BackendSrv`.
```

Comment:
```suggestion
[BackendSrv](https://grafana.com/docs/grafana/latest/packages_api/runtime/backendsrv) handles all outgoing HTTP requests from Grafana. This document explains the high-level concepts used by `BackendSrv`.
```

It may not be obvious to a newcomer what a "data request service" is. Instead of explaining what BackendSrv, I think it's better to explain what it does. Feel free to change my suggestion to something more accurate.